doc/Language/5to6-nutshell.pod6

```diff
@@ -398,6 +398,14 @@ operator rather than a special syntactic form, and thus L<checking for
 existence of elements|#exists> and L<unsetting elements|#delete> is done
 with adverbs.
 
+An array's biggest index is now available with the C<.end> method:
+
+=for code :lang<perl>
+say $#item;    # Perl
+=for code :preamble<no strict;>
+say @item.end; # Raku
+=end item
```

Comment:
You need to delete this; `=for` does not need closing.

Reply:
Ah, sorry. It was a copy-and-paste from the block above. I just changed the code and hoped the Pod would still be valid.